From f1a283dc21a295d31dfb7d37ed67b1359841070c Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 4 Nov 2013 21:35:48 +0000
Subject: [PATCH] builder: Don't use git tree / libguestfs.git to refer to the
 source.

git is just a VCS.

(cherry picked from commit 3ffdddbedf176fa5f12172fb2e3c6bd4b2690f9e)
---
 builder/virt-builder.pod | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/builder/virt-builder.pod b/builder/virt-builder.pod
index 4e33bc8..5eba666 100644
--- a/builder/virt-builder.pod
+++ b/builder/virt-builder.pod
@@ -949,7 +949,7 @@ out the information in this index file.
 
 The templates hosted on libguestfs.org were created using shell
 scripts, kickstart files and preseed files which can be found in the
-libguestfs source tree, in C<libguestfs.git/builder/website>.
+libguestfs source tree, in C<builder/website>.
 
 =head3 Setting up the repository
 
@@ -994,8 +994,7 @@ Also find the fingerprint of your key:
 There are many ways to create the templates.  For example you could
 clone existing guests (see L<virt-sysprep(1)>), or you could install a
 guest by hand (L<virt-install(1)>).  To see how the templates were
-created for virt-builder, look at the scripts in
-C<libguestfs.git/builder/website>
+created for virt-builder, look at the scripts in C<builder/website>
 
 For best results when compressing the templates, use the following xz
 options (see L<nbdkit-xz-plugin(1)> for further explanation):
-- 
1.8.3.1

